packages/blockchain-wallet-v4-frontend/src/modals/Swap/CoinSelection/index.tsx

```diff
@@ -183,11 +137,17 @@ class CoinSelection extends PureComponent<Props> {
                 <CoinAccountListOption
                   account={account}
                   coinModel={coins[account.coin]}
-                  onClick={() =>
-                    this.props.swapActions.changePair(this.props.side, account)
-                  }
+                  onClick={() => {
+                    if (this.props.side === 'BASE') {
+                      this.props.swapActions.changeBase(account)
```

Comment:
should make this an `if/else` instead of two ifs or use `return` `this.props.swapActions...`

Reply:
I've returned it before then.